### PR TITLE
issue-165 Resetting the "direction" to default value upon ending the tour

### PR DESCRIPTION
**Pull request by alex4ndern**
Regarding https://github.com/hakimio/ngx-ui-tour/issues/165

### CODE REVIEW

**Comment by hakimio** [comment]
You still need to:
- update all changelogs
- bump version numbers in all library `package.json` files

**Comment by alex4ndern** [comment]
> You still need to:
> 
> * update all changelogs
> * bump version numbers in all library `package.json` files

Like this?

File: projects/ngx-ui-tour-console/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,10 @@
+<a name="12.0.1"></a>
+
+# [12.0.1](https://github.com/hakimio/ngx-ui-tour) (2024-01-25)
+
+### Fixes
+- Tour is now able to restart even if it was closed right after navigating to the previous step, while the first step is optional and not available
```
Comment (hakimio) [inline projects/ngx-ui-tour-console/CHANGELOG.md:6]: `Fix issue preventing the tour from starting when it was closed after navigating back, while the first step is optional and not available.`

File: projects/ngx-ui-tour-console/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ngx-ui-tour-console",
-  "version": "12.0.0",
+  "version": "12.0.1",
```
Comment (hakimio) [inline projects/ngx-ui-tour-console/package.json:3]: `ngx-ui-tour-core` dependency have to be updated as well:
```json
  "dependencies": {
    "tslib": "^2.0.0",
    "ngx-ui-tour-core": "12.0.1"
  }

```

**Review by hakimio — CHANGES_REQUESTED** [review]

**Comment by alex4ndern** [comment]
Will create a new PR soon

**Comment by hakimio** [comment]
@alex4ndern why did you close this?

**Comment by alex4ndern** [comment]
> @alex4ndern why did you close this?

Wanted to update the Author properly, will reopen a new one soon, sorry

**Comment by alex4ndern** [comment]
https://github.com/hakimio/ngx-ui-tour/pull/168
here is the new one

**Pull request closed without merging**